packages/lesswrong/components/spotlights/SpotlightItem.tsx

```diff
@@ -230,6 +241,8 @@ export const SpotlightItem = ({classes, spotlight, showAdminInfo, hideBanner, re
   // But, also, the real proper fix here is to integrate continue reading here.
   const firstPostUrl = spotlight.firstPost && postGetPageUrl(spotlight.firstPost, false, spotlight.documentType === "Sequence" ? spotlight.documentId : undefined)
 
+  const duration = spotlight.duration || DEFAULT_ACTIVE_DAYS
```

Comment:
nit: `??` (assuming we want `0` to be a meaningful value here, which, probably?)

Reply:
Also I don't think we should import `DEFAULT_ACTIVE_DAYS` from the cron file, which lives in `server`; we may want to stick that in a `helpers` or `constants` file in the collection folder